src/net.h

```diff
@@ -284,7 +284,7 @@ class CBanEntry
     ADD_SERIALIZE_METHODS;
 
     template <typename Stream, typename Operation>
-    inline void SerializationOp(Stream& s, Operation ser_action, int nType, int nVersion) {
+    inline void SerializationOp(Stream& s, Operation ser_action, int nType, int _nVersion) {
```

Comment:
The same code is in https://github.com/bitcoin/bitcoin/blob/master/src/qt/walletmodel.h#L68-L78.